binding/form_mapping.go

```diff
@@ -170,6 +170,10 @@ func setByForm(value reflect.Value, field reflect.StructField, form map[string][
 }
 
 func setWithProperType(val string, value reflect.Value, field reflect.StructField) error {
+	if value.Kind() != reflect.String {
```

Comment:
This change is not explained by the PR description.
Please update PR description with explanation what cases have you fixed and why.

Reply:
If it is a string type, no spaces are removed, and the user data is not modified here.

Reply:
Please document your changes in the PR's description. Just add information about what is changed and why.